src/components/views/dialogs/BaseDialog.js

```diff
@@ -36,6 +36,9 @@ export default React.createClass({
 
     propTypes: {
         // onFinished callback to call when Escape is pressed
+        // Take a boolean which is true is the dialog was dismissed
+        // with a positive / confirm action or false if it was
+        // cancelled (from BaseDialog, this is always false).
```

Comment:
Also, onFinished is never called with a truthy value, so the next comment is not true.

Reply:
Oh, well if the BaseDialog has no way to provide an affirmative response, this makes sense. Thanks for clarifying.